crates/divan_compat/examples/README.md

```diff
@@ -0,0 +1,3 @@
+# Example benchmarks
+
+Mostly copied from [divan's examples](https://github.com/nvzqz/divan/tree/main/examples).
```

Comment:
As discussed, we should copy the example files from divan and use cfg not codspeed for the benches not yet supported.
Probably adding a header at the top of each file mentioning it's from divan will help